policies/general/index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: OpenSSL General Policies
+---
+These are the general policies and procedures established by the OMC
```

Comment:
Colon at the end?

Reply:
We don't usually do that.  But a period at the very least, yeah

Reply:
Done